### PR TITLE
Feat/coin pile

### PR DESCRIPTION
**Pull request by janhenrikfock**
Merged Main into the Feature-Branch. Is this Feature ready to merge @jamarob ?

### CODE REVIEW

**Review by jamarob — COMMENTED** [review]
Hm.. don't know 😅 this was just something I tried out, but if you think it is useful, use it!

**Comment by janhenrikfock** [comment]
Since Leon seemed to like it i would use it.

File: src/components/CoinPile/CoinPile.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components'
+import coin from '../../assets/qcoin.png'
+
+export default function CoinPile({ coins }) {
+  return (
+    <StyledCoinPile>
+      {[...new Array(coins)].map(() => (
+        <StyledCoin src={coin} />
```
Comment (mpagels) [inline src/components/CoinPile/CoinPile.js]: Idk but do we need here a key prop?

**Review by mpagels — APPROVED** [review]
Good job, found only one little thing.

File: src/components/CoinPile/CoinPile.js (hunk 1 above)
Comment (jamarob) [reply src/components/CoinPile/CoinPile.js]: Good catch! As one coin is like the others a key is not really needed, but react will complain anyway.

**Review by mpagels — APPROVED** [review]

**Comment by janhenrikfock** [comment]
If there aren't any further objections, i will merge tommorrow.

**Pull request merged**